alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
+                --padding: 6px;
+            }
+            
+            .container {
+                background-color: var(--less-red);
+                border-width: 0;
+                border-left: var(--border-width) solid var(--very-red);
+                border-bottom-right-radius: var(--border-radius);
+                border-top-right-radius: var(--border-radius);
+                padding: var(--padding);
+            }
+            
+            .container [dir='rtl'] {
+		        border-left-width: 0;
+		        border-right: var(--border-width) solid var(--very-red);
+		        border-bottom-right-radius: 0;
+		        border-top-right-radius: 0;
+		        border-bottom-left-radius: var(--border-radius);
+		        border-top-left-radius: var(--border-radius);
+	        }
+        </style>
+        
+        <div class="container">
+            <content></content>
+        </div>
+    </template>
+    
+    <script>
+        Polymer({
```

Comment:
Note: one of the behaviors implemented in [react-vui-alerts](https://github.com/Brightspace/react-valence-ui-alerts/blob/master/src/error.js) is to announce the alert.  In that case it was done by specifying the Aria `role="alert"`.  I suspect we still want inline alerts to be announced.